Makefile

```diff
@@ -112,16 +124,30 @@ manifest-templator: $(GO)
 
 docker-build: docker-build-operator docker-build-registry
 
-docker-build-operator: manager manifest-templator
-	$(OCI_BIN) build -f build/operator/Dockerfile -t $(IMAGE_REGISTRY)/$(OPERATOR_IMAGE):$(IMAGE_TAG) .
+docker-build-operator:
+ifeq ($(OCI_BIN),podman)
+	$(MAKE) build-multiarch-operator-podman
+else ifeq ($(OCI_BIN),docker)
+	$(MAKE) build-multiarch-operator-docker
+else
+	$(error Unsupported OCI_BIN value: $(OCI_BIN))
+endif
 
 docker-build-registry:
 	$(OCI_BIN) build -f build/registry/Dockerfile -t $(IMAGE_REGISTRY)/$(REGISTRY_IMAGE):$(IMAGE_TAG) .
 
 docker-push: docker-push-operator docker-push-registry
 
 docker-push-operator:
-	$(OCI_BIN) push ${TLS_SETTING} $(IMAGE_REGISTRY)/$(OPERATOR_IMAGE):$(IMAGE_TAG)
+ifeq ($(OCI_BIN),podman)
+	podman manifest push ${TLS_SETTING} ${OPERATOR_IMAGE_TAGGED} ${OPERATOR_IMAGE_TAGGED}
+else ifeq ($(OCI_BIN),docker)
+ifeq ($(words $(PLATFORM_LIST)), 1)
+	docker push ${TLS_SETTING} ${OPERATOR_IMAGE_TAGGED}
+endif
```

Comment:
please try to think about a makefile target structure that can be simpler wrt to multi platform and one platform
the logic is split here and on docker-build-operator, it will be better to have it concentrated if possible (i understand it might be not possible because there are stages)

Reply:
For docker buildx, we need to combine both build and push in a single command because the images are not stored locally by default, unlike with docker build. As a result, we don't need a separate docker push command when working with multi-platform builds. To simplify the logic and maintain consistency, we could consider adding the --push option to docker build as well. What do you think @oshoval

Reply:
might be interesting to try if you think it will simplify it please

Reply:
Done.